libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -904,6 +904,8 @@ string ABIFunctions::abiEncodingFunctionStruct(
 				<#members>
 				{
 					// <memberName>
+					<preprocess>
```

Comment:
This used to be in old template `memberTempl` that was rendered to `<encode>`, i.e. I just pulled these two lines out from the encode template.